src/main/java/me/dinowernli/grpc/prometheus/Configuration.java

```diff
@@ -66,12 +84,24 @@ public CollectorRegistry getCollectorRegistry() {
     return latencyBuckets;
   }
 
+  /** Returns the configured list of headers to be used as labels. */
+  public List<String> getLabelHeaders() {
```

Comment:
Why do we need to expose both the original label headers and the sanitized label headers? Could we get rid of one of the two?

Reply:
We need this so that ServerMetrics/ClientMetrics is able to also know the original header names to look for them in each request.